bin/si-smoketest/tests/get_head_changeset.ts

```diff
@@ -0,0 +1,10 @@
+import assert from "node:assert";
+import { SdfApiClient } from "../sdf_api_client.ts";
+
+export default async function get_head_changeset(sdfApiClient: SdfApiClient) {
+  const data = await sdfApiClient.listOpenChangeSets();
```

Comment:
I don't necessarily love the separation of the URL and payload from the test (e.g. inherently coupled data living in two different places)... but I can get over that 😇

Reply:
This is more of a smoke test for the smoke tester, it's made to make sure the simplest get request can execute, but other places may nee to list open changesets - this test can got away but the helper in the client would stay

Reply:
@johnrwatson and I brainstormed on this little bit:
```
interface API_REQUEST {
    path: string,
    method: string,
    headers: Record<string, string>
}

const APIS<string, API_REQUEST> = {
    open_changeset: { path: '/change_set/list_open_change_sets', method: 'GET', headers: { ... } },
    schema_variants: { path: '/v2/workspaces/${workspaceId}/change-sets/${changeSetId}/schema-variants', method: 'GET'},
}

await sdfApiClient.call(APIS.open_changeset, payload)
```

Reply:
Dreamy, let me implement

Reply:
there's already a fetch method on the client

Reply:
We either implement the methods for each endpoint or this interface and update fetch, otherwise there's like 2 abstractions for the same thing

Reply:
And since the methods are already there (and this PR is blocking me merging my test a bit 😅 ) i'd suggest we leave it as it is

Reply:
(I'm merging this just to get the code locally but this can be talked about/implement in a folloup PR, I'm don't wanna kill the conversation by merging this pr)